### PR TITLE
Update the vote a core developer: PPMC => PMC

### PR DESCRIPTION
**Pull request by git-hulk**
- Update PPMC to PMC after graduation
- Remove incubator keyword

### CODE REVIEW

File: community/vote-a-core-developer.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ receive a follow-up message with the next steps for
 establishing you as a committer.
 ```
 
-For inviting the PPMC member, the mail template can refer [this](https://incubator.apache.org/guides/ppmc-offer.txt).
```
Comment (tisonkun) [inline community/vote-a-core-developer.md:213]: Perhaps change the link also. I'm not sure which one can be as a replacement, though.

File: community/vote-a-core-developer.md [hunk 2]
```diff
@@ -105,22 +105,22 @@ Then I'm going to invite XXXXXX to join us.
...
-Note: If it is not passed, the result is "The vote for "XXXXXX as a Kvrocks [committer|PPMC member]" has FAILED and closed now."
+Note: If it is not passed, the result is "The vote for "XXXXXX as a Kvrocks [committer|PMC member]" has FAILED and closed now."
 
-## Newly added PPMC member notification email
+## Newly added PMC member notification email
 
-This step is only processed for the PPMC member that passed the vote. If the election is Committer, this step is skipped and not executed.
+This step is only processed for the PMC member that passed the vote. If the election is Committer, this step is skipped and not executed.
 
 Voting initiators need to send notification emails to Board's mailing group and wait at least 72 hours; email owners send `private@incubator.apache.org` and CC `private@kvrocks.apache.org`; Board will analyze compliance Until there is no doubt.
```
Comment (tisonkun) [inline community/vote-a-core-developer.md:114]: Now we should send to `board@apache.org`.

File: community/vote-a-core-developer.md (hunk 1 above)
Comment (git-hulk) [reply community/vote-a-core-developer.md:213]: Yes, I didn’t find the right link, so leave it as it’s in this PR.

File: community/vote-a-core-developer.md (hunk 2 above)
Comment (git-hulk) [inline community/vote-a-core-developer.md:114]:
```suggestion
Voting initiators need to send notification emails to Board's mailing group and wait at least 72 hours; email owners send `board@apache.org` and CC `private@kvrocks.apache.org`; Board will analyze compliance Until there is no doubt.
```

**Review by tisonkun — APPROVED** [review]

**Pull request merged**